alg/teca_surface_pressure_estimate.cxx

```diff
@@ -0,0 +1,262 @@
+#include "teca_surface_pressure_estimate.h"
+
+#include "teca_mesh.h"
+#include "teca_array_collection.h"
+#include "teca_variant_array.h"
+#include "teca_metadata.h"
+#include "teca_array_attributes.h"
+
+#include <algorithm>
+#include <iostream>
+#include <string>
+#include <set>
+#include <cmath>
+
+#if defined(TECA_HAS_BOOST)
+#include <boost/program_options.hpp>
+#endif
+
+//#define TECA_DEBUG
+
+namespace internal
+{
+template <typename num_t, typename coord_t>
+void compute_surface_pressure(size_t n_elem, const num_t *tas,
+    const num_t *psl, coord_t *z, num_t *ps)
+{
+    num_t cp = 1.0; // kJ/kg.K
```

Comment:
cp and cv should be in units of J/Kg/K.  This would matter for `gam` since the units cancel, but it does matter for the `GAM * zi` parts.

I suggest `cp=1004.` and `cv = 718.` as the values used